contracts/src/osp/OneStepProverMath.sol

```diff
@@ -291,7 +291,12 @@ contract OneStepProverMath is IOneStepProver {
                 // rotr
                 res = rotr32(a, b);
             } else {
-                res = uint32(genericBinOp(a, b, opcodeOffset));
+                (uint256 computed, bool err) = genericBinOp(a, b, opcodeOffset);
```

Comment:
computed should be a `uint64` here as that's what genericBinOp returns

Reply:
Fixed